website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -167,6 +168,35 @@ Avoid using different density settings for parent and child rows.
 
 !!!
 
+### Expand/Collapse All Button
+The Expand/Collapse All button allows users to expand or collapse all rows, including nested rows. It provides quick access to more content but may impact readability when content is long or detailed.
+
+### Interactions
+
+#### Default state
+
+The Advanced Table supports any combination of expanded or collapsed rows on load. The button reflects the initial state.
+
+![Expand all button with an expand icon.](/assets/components/table/advanced-table/expandable-rows-expand-all.png)
+<Doc::ImageCaption @text="Expand All if any rows are collapsed."/>
```

Comment:
Nit: I would probably just say "Expand all rows" for simplicity. (vs. "Expand All if any rows are collapsed")

Reply:
I think I have better suggestions for these... Working on it...